Sources/FoundationNetworking/URLSession/libcurl/MultiHandle.swift

```diff
@@ -281,6 +281,12 @@ fileprivate extension _EasyHandle {
             return NSURLErrorTimedOut
         case (CFURLSessionEasyCodeOPERATION_TIMEDOUT, _):
             return NSURLErrorTimedOut
+        case (CFURLSessionEasyCodeSSL_CACERT, _):
+            return NSURLErrorServerCertificateUntrusted
+        case (CFURLSessionEasyCodePEER_FAILED_VERIFICATION, _):
+            return NSURLErrorServerCertificateWrongHost
```

Comment:
Hmm, CURLE_SSL_CACERT is just an alias for CURLE_PEER_FAILED_VERIFICATION, they are same.